hack/codegen.sh

```diff
@@ -67,6 +67,8 @@ statik-gen operator config/operator
 $CONTROLLER_GEN crd:crdVersions=${CRD_VERSION} webhook paths="./pkg/apis/fake/..." output:crd:artifacts:config=config/internal/fake-operator/crd/bases output:webhook:artifacts:config=config/internal/fake-operator/webhook
```

Comment:
```suggestion
$CONTROLLER_GEN crd:crdVersions=${CRD_VERSION} paths="./pkg/apis/fake/..." output:crd:artifacts:config=config/internal/fake-operator/crd/bases
```
Since you are moving the webhook generate command to line 70 -71, this can be simplified

Reply:
thx